RELEASE-NOTES.md

```diff
@@ -1,5 +1,50 @@
 # Prefect Release Notes
 
+## Release 2.7.11
+
+### Exciting New Features 🎉
+- Add refactor of results into a standalone artifact table — https://github.com/PrefectHQ/prefect/pull/8164
+- Add Prefect Cloud Events schema and clients. — https://github.com/PrefectHQ/prefect/pull/8357
+
+### Enhancements
+- Update pinned pip version in Docker images — https://github.com/PrefectHQ/prefect/pull/8289
+- Update default task run name to exclude hash of task key — https://github.com/PrefectHQ/prefect/pull/8292
+- Add functionality that updates preinstalled packages in Docker images — https://github.com/PrefectHQ/prefect/pull/8288
+- Add `PREFECT_LOGGING_ORION_WHEN_MISSING_FLOW` setting to handle error raised when logger called outside of flow run context — https://github.com/PrefectHQ/prefect/pull/8311
+
+### Fixes
+- Fix cancellation bug — https://github.com/PrefectHQ/prefect/pull/8315
+- Fix Kubernetes job watch exit when no timeout given — https://github.com/PrefectHQ/prefect/pull/8350
+- Fix default value for priority on `WorkQueue` core schema — https://github.com/PrefectHQ/prefect/pull/8373
```

Comment:
What does this mean if I'm a user? This adds a default value for priority for compatibility with previous client versions, right?

Reply:
We should move this to experimental